packages/table/src/cell/formats/jsonFormat.tsx

```diff
@@ -30,26 +29,25 @@ export interface IJSONFormatProps extends ITruncatedFormatProps {
     stringify?: (obj: any) => string;
 }
 
-@PureRender
```

Comment:
that doesn't sound right, this is a simple stateless component without any lifecycle hooks -- can you dig in to why it's not re-rendering, probably something further up the tree